File: readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <ctype.h>
#include <string.h>
#define isalphap(c) (isalpha(c) && 1)
int letters(string message);

int main(void)
{
    string message = get_string("Text: "); // getting an input from user
    printf("letters is %i\n",letters(message));
}

int letters(string message) // searching for letters in message
{
    int n = 0;
    for (int i = 0; i < strlen(message); i++)
    {
        if (isalphap(message[i]))
        {
            n++;
        }
    }
    return n;
}

File: sentimental-readability/readability.py
from cs50 import get_string

string = get_string("Text: ")

w = 1 # w =1
s = 0

for letter in string: # getting l,s
    if (letter == " "):
        w += 1
    if (letter == "!" or letter == "."): # number of sentences
        s += 1

l = len(string) #number of letters in string
L = l/100
S = s/100 *l
eq =0.0588 * L - 0.296 * S - 15.8

print(eq)

